src/scripting/lua_interface.cc

```diff
@@ -60,6 +60,7 @@ LuaInterface::LuaInterface() {
 	open_lua_library(lua_state_, LUA_MATHLIBNAME, luaopen_math, true);
 	open_lua_library(lua_state_, LUA_DBLIBNAME, luaopen_debug, true);
 	open_lua_library(lua_state_, LUA_COLIBNAME, luaopen_coroutine, true);
+	open_lua_library(lua_state_, LUA_IOLIBNAME, luaopen_io, true);
```

Comment:
Please revert this, it's a security vulnerability and should only be active when the user really really wants it

Reply:
will do so in the final version. this was needed to create the initial infrastructure and I'll keep it until Im am sure I do not need to modifiy this again.

Reply:
would commenting it out be sufficient?